rootio/tchain.go

```diff
@@ -0,0 +1,115 @@
+package rootio
+
+type tchain struct {
+	trees []Tree
+}
+
+func (tchain) Class() string {
+	return "TChain"
+}
+
+// Chain returns a tchain that is the concatenation of all the input Trees.
+func Chain(trees ...Tree) tchain {
+	var t tchain
+	t.trees = append(t.trees, trees...)
+	return t
+}
+
+func (t tchain) Entries() int64 {
+	var v int64 = 0
+	for i := range t.trees {
```

Comment:
here, we aren't really interested in the index into that slice of trees: we just use the indexed value.
so we can use this form of `range`:
```go
for _, tree := range t.trees {
    v += tree.Entries()
}
```